wcfsetup/install/files/lib/system/importer/AbstractAttachmentImporter.class.php

```diff
@@ -108,6 +109,32 @@ protected function fixEmbeddedAttachments($message, $oldID, $newID) {
 			
 			return $message;
 		}
+		else {
+			return \preg_replace_callback(
+				'~<woltlab-metacode data-name="attach" data-attributes="(?<attributes>[^"]+)">~',
+				function (array $matches) use ($oldID, $newID): string {
```

Comment:
```suggestion
				static function (array $matches) use ($oldID, $newID): string {
```

Otherwise this will fail after the merge to master.